src/Bridges/Nette/MailSender.php

```diff
@@ -39,6 +39,7 @@ public function __construct(Nette\Mail\IMailer $mailer, string $fromEmail = null
 	 */
 	public function send($message, string $email): void
 	{
+		assert(\class_exists(Nette\Mail\Message::class));
```

Comment:
it's better to add nette/mail as a dev dependency

Reply:
Added.